From ec4c7b5459de395880221217471b91a55156de5b Mon Sep 17 00:00:00 2001
From: Paul Cercueil <paul@crapouillou.net>
Date: Fri, 8 Dec 2023 12:51:31 +0100
Subject: [PATCH] crypto: Fix build on MIPS

Signed-off-by: Paul Cercueil <paul@crapouillou.net>
---
 crypto/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/crypto/CMakeLists.txt b/crypto/CMakeLists.txt
index 7efb143..4b2c21c 100644
--- a/crypto/CMakeLists.txt
+++ b/crypto/CMakeLists.txt
@@ -970,6 +970,8 @@ elseif(HOST_ARM)
 	target_include_directories(crypto_obj PRIVATE bn/arch/arm/)
 elseif(HOST_I386)
 	target_include_directories(crypto_obj PRIVATE bn/arch/i386/)
+elseif(HOST_MIPS)
+	target_include_directories(crypto_obj PRIVATE bn/arch/mips)
 elseif(HOST_MIPS64)
 	target_include_directories(crypto_obj PRIVATE bn/arch/mips64)
 elseif(HOST_POWERPC)
-- 
2.42.0

